jwtBearer.go

```diff
@@ -78,17 +78,7 @@ func (c *jwtBearer) newAccessToken() error {
 		c.setErr(err)
 	}()
 	// Create JWT
-	token := jwt.NewWithClaims(
-		jwt.SigningMethodRS256,
-		jwt.StandardClaims{
-			Issuer:    c.consumerKey,
-			Audience:  c.authServerURL,
-			Subject:   c.username,
-			ExpiresAt: time.Now().Add(c.tokenDuration).UTC().Unix(),
-		},
-	)
-	// Sign JWT with the private key
-	signedJWT, err := token.SignedString(c.rsaPrivateKey)
+	signedJWT, err := c.JWT()
```

Comment:
I like the way this refactor worked out! :rocket: Makes sense to make this its own separate method and call it here.